Core.DomainModel/GDPR/Read/DataProcessingRegistrationReadModel.cs

```diff
@@ -59,5 +59,8 @@ public DataProcessingRegistrationReadModel()
         public YesNoUndecidedOption? IsOversightCompleted { get; set; }
 
         public string ContractNamesAsCsv { get; set; }
+        public int? LastChangedById { get; set; }
+        public string LastChangedByName { get; set; }
```

Comment:
Remember to add indexes, so that filtering speed is improved

Reply:
Example: 
![image](https://user-images.githubusercontent.com/13116889/173325556-c319f018-73d4-44b6-a335-7aef3bfc76a8.png)

**And then the migration should be recreated ofc**